src/app/event/event.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnInit } from '@angular/core';
+import { ProgramService } from '../Services/programe.service';
+import { NgForm } from '@angular/forms';
+import { Event } from '../model/event';
+declare var AWS: any;
+
+@Component({
+  selector: 'app-progrmme',
+  templateUrl: './event.component.html',
+  styleUrls: ['./event.component.css','../../assets/css/loading-animation.css']
+})
+export class EventComponent {
+
+  selectedFiles: FileList;
+  eProg = [];
+  SelectedProgId: number;
+  isNotUploaded = true;
+  isUploading = false;
+  isSubmiting = false;
+  pdfUrl = ''; 
+
+  constructor(private progService: ProgramService) {
+
+    //   progService.getAll().subscribe(
+    //     (data) => {
+    //       console.log(data);
+    //       this.eProg = data;
+    //     }
+    //  );
+  }
+
+
+  selectFile(event) {
+    this.selectedFiles = event.target.files;
+  }
+
+  onUpload (value: any) {
+     
+    console.log('up load fun ', value);
+    this.isUploading = true ;
+    const file = this.selectedFiles.item(0);
+    if (file) {
+        AWS.config.update({
+            'accessKeyId': 'AKIAILGGQK25JKQI6Q5A',
```

Comment:
access key should come from constant files
because we have to configure from aws account